docs/api-docs/storefront/graphql/graphql-variants.md

```diff
@@ -0,0 +1,814 @@
+
+<!--
+type: tab
+title: Variant entity ID
+-->
+
+
+```graphql title="Get a variant using the variant entity ID" lineNumbers
+# This query retrieves 1 variant. 
+
+query {
+  site {
+    product (variantEntityId: 27098) {
+      # fields on the Product object type  
+    }
+  }
+}
+```
+
+<!--
+type: tab
+title: Variant option value entity ID
+-->
+
+```graphql title="Get a variant using variant option values" lineNumbers
+# This query retrieves 1 variant.
+# You must specify the entity ID for the value of each variant option.
+
+query {
+  site {
+    product (
+      entityId: 113       # product entity ID
+      optionValueIds: [{optionEntityId: 116, valueEntityId: 108} {optionEntityId: 126, valueEntityId: 129}]
+    ) {
+      # fields on the Product object type
+    }
+  }
+}
+
+```
+<!--
+type: tab
+title: Variant SKU
+-->
+
+```graphql title="Get a variant using the variant sku" lineNumbers
+# This query retrieves 1 variant. 
+
+query {
+  site {
+    product (sku: "variant-sku") {
+      # fields on the Product object type 
+    }
+  }
+}
+```
+
+<!-- type: tab-end -->
+
+Specifying a variant identifier for the `product` field returns variant information overlaid on the Product object. For example, if the variant has a different image, dimensions, SKU, or price, than the product, the variant's info will be returned. This allows you to directly query a variant.
+
+## Get variant identifiers
+
+You can query identifying info for variants. The following query retrieves identifying info for the specified variant:
+
+<!--
+type: tab
+title: Query
+-->
+
+```graphql title="Example query: Get basic info for a variant" lineNumbers
+query {
+  site {
+    product (entityId: 113) {
+      variants (entityIds: [127]) {
+        edges {
+          node {
+            id
+            entityId
+            sku  
+            upc
+            mpn
+          }
+        }
+      }   
+    }
+  }
+}
+```
+
+<!--
+type: tab
+title: Response
+-->
+
+```json title="Example response: Get basic info for a variant" lineNumbers
+{
+  "data": {
+    "site": {
+      "product": {
+        "variants": {
+          "edges": [
+            {
+              "node": {
+                "id": "VmFyaWFudDoxMjc=",
+                "entityId": 127,
+                "sku": "ABC-1111-PI-SM-CO-RE",
+                "upc": "123346",
+                "mpn": "9876543210"
+              }
+            }
+          ]
+        }
+      }
+    }
+  }
+}
+```
+<!-- type: tab-end -->
+
+## Get variant prices and dimensions
+
+You can query the prices and dimensions for variants. The following query retrieves prices and dimensions for the specified variant:
+
+<!--
+type: tab
+title: Query
+-->
+
+```graphql title="Example query: Get prices and dimensions for a variant" lineNumbers
+# This query uses fragments. For more, see https://graphql.org/learn/queries/#fragments.
```

Comment:
```suggestion
# This query uses fragments. To learn more about fragments, see GraphQls' documentation segment on [fragments](https://graphql.org/learn/queries).
```

Reply:
I used the copyedit, though the [text](link) doesn't work in code comments